### DISCUSSION TITLE
UserScript to bring back the old feed

### DISCUSSION THREAD

**Discussion by Gerrit0** [community/community#37694] [Other Feature Feedback, Questions, & Ideas]
This UserScript will remove the "For You" section in favor of the old feed. It will additionally cache your feed, immediately showing results rather than waiting for the 3-5 seconds that it takes GitHub's feed fetch to come back.

https://github.com/Gerrit0/old-github-feed

**Comment by yan12125** [comment]
It works, thanks!

**Comment by TeodorVecerdi** [comment]
Thanks!

**Comment by siddhpant** [comment]
Thanks, it works for me too!

**Comment by jevinskie** [comment]
You’ve made my day, this works great on mobile Safari! ❤️😘

**Reply by jevinskie** [reply]
I found that replacing `const dashboard = document.getElementById('dashboard');` with `const dashboard = document.getElementsByClassName('js-feed-container')[0];` it keeps my "Recent Repositories", "Recent Activities", and "Your teams" on "mobile" (really just narrow width browser window).

Before:
<img width="686" alt="before" src="https://user-images.githubusercontent.com/187848/198837957-0f7b6ddc-970f-4bed-b021-3d2fdb79c39f.png">

After:
<img width="686" alt="after" src="https://user-images.githubusercontent.com/187848/198837962-0c18e542-9361-4df3-ad9e-c24c0e18986e.png">

**Reply by Gerrit0** [reply]
Thanks for the bug report! I've updated it to use `dashboard-feed-frame`, which seemed more convenient.

**Reply by jevinskie** [reply]
Great thank you! I also updated it to match /dashboard. Any ideas about getting the more button to work? The AJAX request goes through fine but the callback in dashboard.js doesn’t seem to replace the  `<form>` like it should.

https://github.com/Gerrit0/old-github-feed/compare/main...jevinskie:old-github-feed:main

**Reply by Gerrit0** [reply]
@jevinskie that's because the MutationObserver is used to prevent GitHub's JS from overwriting it. 0.3 should fix it.

**Reply by jevinskie** [reply]
0.3 seems to work perfectly, many thanks! 🔥

**Comment by akshitagupta15june** [comment]
"Your feed" section will be wonderful as we will be able to see the updates in our own repositories also the stars, followers as well as a forks.

**Comment by xSnowHeadx** [comment]
Nice work! Thanks for sharing it.
